src/ResourceManager/Sql/Commands.Sql/help/Add-AzureRmSqlInstanceTransparentDataEncryptionCertificate.md

```diff
@@ -38,7 +38,7 @@ PS C:\> Add-AzureRmSqlManagedInstanceTransparentDataEncryptionCertificate -Resou
 The credentials, account, tenant, and subscription used for communication with Azure.
 
 ```yaml
-Type: Microsoft.Azure.Commands.Common.Authentication.Abstractions.IAzureContextContainer
```

Comment:
Please upgrade to the latest platyPS and run with the -UseFullTypeName parameter: https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module